sqlconnect/internal/redshift/tableadmin.go

```diff
@@ -0,0 +1,27 @@
+package redshift
+
+import (
+	"context"
+	"fmt"
+	"github.com/rudderlabs/sqlconnect-go/sqlconnect"
+)
+
+func (db *DB) ListColumns(ctx context.Context, relation sqlconnect.RelationRef) ([]sqlconnect.ColumnRef, error) {
```

Comment:
[SVV_COLUMNS](https://docs.aws.amazon.com/redshift/latest/dg/r_SVV_COLUMNS.html) appears to be a more straightforward approach with the added bonus that it supports external tables too

Reply:
updated